app/views/UnloadingGuidanceView.scala.html

```diff
@@ -37,7 +37,7 @@
         <p class="govuk-body">@messages("unloadingGuidance.para", mrn)</p>
 
         <p class="govuk-body">@messages("unloadingGuidance.pdf.preLinkText")
-        <a id="download" class="govuk-link" href=@routes.UnloadingPermissionPDFController.getPDF(arrivalId)>@messages("unloadingGuidance.pdf.link")</a>
+        <a id="download" class="govuk-link" href="???">@messages("unloadingGuidance.pdf.link")</a>
```

Comment:
To do?

Reply:
Yep, we dont have the P5 endpoints for this yet and wanted to make that clear :)

Reply:
Yep, we dont have the P5 endpoint for this yet and wanted to make that clear :)